migrations/20190212102119_cards.js

```diff
@@ -2,9 +2,10 @@
 exports.up = function(knex, Promise) {
   return knex.schema.createTable('cards', (table) => {
     table.increments().primary();
+    table.string('title').notNullable();
```

Comment:
You shouldn't modify migration files. You should make a new migration instead.

Reply:
Made the change in a new commit.

Reply:
If you made changes in a new commit, that should mean that these changes are removed from this commit and don't show up in a diff

Reply:
I see in your other commit you did a `createTable`, what you want to do is leave this commit as is for the `createTable` and your next migration would be to modify the table columns that you want to change.